osu.ElasticIndexer/UnrunnableProcessor.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Threading;
+using osu.Server.QueueProcessor;
+
+namespace osu.ElasticIndexer
+{
+    public class UnrunnableProcessor : QueueProcessor<ScoreItem>
```

Comment:
Is this just meant as safety to ensure that all the local usages never run the queue?

I don't know how beneficial this is, but maybe okay? I'd probably inherit `IndexQueueProcessor` instead though.

Reply:
Some (most?) of the commands only need access to the queue but should not be able to process anything from it and there wasn't a way to get access to the queue-related methods without instantiating `QueueProcessor`.